chat-components/src/androidTest/java/com/pubnub/components/MemberListTest.kt

```diff
@@ -79,7 +80,7 @@ class MemberListTest : BaseTest() {
 
     @Suppress("UNCHECKED_CAST")
     @Test
-    fun whenMemberPagingDataWillBePassed_thenItWillBeShown() = runBlockingTest {
+    fun whenMemberPagingDataWillBePassed_thenItWillBeShown() = runTest(UnconfinedTestDispatcher()) {
```

Comment:
Why do we sometimes use that Dispatcher (explicitly)?

Reply:
Not needed now, all tests should run on StandartTestDispatchers, I'll check that